packages/date-picker/src/panel/date.vue

```diff
@@ -565,6 +626,59 @@
         }
       },
 
+      canSelectPrevMonth() {
```

Comment:
Computeds to check whether prev/next month and year can be navigated to, in the case where `disabledDate` is not defined, but `minimum` and/or `maximum` is